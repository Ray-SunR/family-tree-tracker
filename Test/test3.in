! C
! B
! A
+ A D
+ B E
+ B F
+ C G
+ C H
? E
? H
? G
? F
+ I C
? I
p
