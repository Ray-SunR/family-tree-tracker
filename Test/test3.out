A,B,E
A,B,C,H
A,B,C,G
A,B,F
Failed
Not found
A is parent of B,D
B is parent of C,E,F
C is parent of G,H
